public/enhanced-table-vis-controller.js

```diff
@@ -306,7 +306,7 @@ module.controller('EnhancedTableVisController', function ($scope, Private, confi
   };
 
   const isInt = (item) => {
-    return parseInt(item).toString() == item;
+    return /^[0-9]+$/.test(item);
```

Comment:
It would be nice to support leading and trailing spaces.
So can you put `/^ *[0-9] *$/` ?
IMHO, in previous versions, this was supported.
I don't want that `' 2 '` is interpreted as a label.

Reply:
Absolutely. I didn't mean to remove the trailing spaces. I'll fix it right now.